boards/stm32c0116-dk/dist/openocd.cfg

```diff
@@ -0,0 +1,7 @@
+set WORKAREASIZE 0x2000
+source [find interface/stlink-v2.cfg]
+source [find target/stm32c0x.cfg]
+
+#reset_config srst_only
```

Comment:
Can be dropped?

Reply:
Looking into this, the lines-
```
source [find target/stm32c0x.cfg]
reset_config srst_only
$_TARGETNAME configure -rtos auto
```
are included in https://github.com/RIOT-OS/RIOT/blob/master/boards/common/stm32/dist/stm32c0.cfg so I'm betting they can be removed from the various C0 board openocd.cfg files?